<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMCertificateDialog</class>
 <widget class="QDialog" name="QMCertificateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nachweise verwalten</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Typ</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cbType">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::NoInsert</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="tbResetFilter">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/icons/edit-reset.svg</normaloff>:/icons/icons/edit-reset.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="leName"/>
     </item>
     <item row="1" column="3">
      <widget class="QToolButton" name="tbFilter">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/icons/view-filter.svg</normaloff>:/icons/icons/view-filter.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableView" name="tvFiles">
     <property name="editTriggers">
      <set>QAbstractItemView::SelectedClicked</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QToolButton" name="tbAdd">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/icons/list-add.svg</normaloff>:/icons/icons/list-add.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbRemove">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/icons/list-remove.svg</normaloff>:/icons/icons/list-remove.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbShowCertificate">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/icons/document-preview-archive.svg</normaloff>:/icons/icons/document-preview-archive.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QMCertificateDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QMCertificateDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbAdd</sender>
   <signal>clicked()</signal>
   <receiver>QMCertificateDialog</receiver>
   <slot>addCertificate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>732</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbRemove</sender>
   <signal>clicked()</signal>
   <receiver>QMCertificateDialog</receiver>
   <slot>removeCertificate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>442</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbShowCertificate</sender>
   <signal>clicked()</signal>
   <receiver>QMCertificateDialog</receiver>
   <slot>showCertificate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>442</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbFilter</sender>
   <signal>clicked()</signal>
   <receiver>QMCertificateDialog</receiver>
   <slot>updateFilter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbResetFilter</sender>
   <signal>clicked()</signal>
   <receiver>QMCertificateDialog</receiver>
   <slot>resetFilter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addCertificate()</slot>
  <slot>removeCertificate()</slot>
  <slot>showCertificate()</slot>
  <slot>updateFilter()</slot>
  <slot>resetFilter()</slot>
 </slots>
</ui>
